Resources/config/security.xml

```diff
@@ -15,7 +15,7 @@
             <argument type="service" id="fos_user.user_manager" />
         </service>
 
-        <service id="fos_user.security.login_manager" class="%fos_user.security.login_manager.class%">
+        <service id="fos_user.security.login_manager" class="%fos_user.security.login_manager.class%" public="true">
```

Comment:
AFAICT, this one does not need to be public (we only inject it in a listener)

Reply:
So we need to decide what to do here. Because some *users* are using these publicly (e.g.  https://stackoverflow.com/questions/12672169/how-to-restfully-login-symfony2-security-fosuserbundle-fosrestbundle).

Do we make them all private and bump a new major?

Reply:
Well, they would be affected only once they use Symfony 4. In symfony 3.4, they will see a deprecation message saying that the service will become private when switching to SF4.
Btw, this is why it is the *only* time in history where we can safely migrate to private services (by relying on the SF default visibility change to perform the migration).

People wanting to use the LoginManager in their controller should switch to injecting it instead of using the container IMO (not sure we have the autowiring alias for our LoginManagerInterface, but we could add it)